snowpack/src/sources/local.ts

```diff
@@ -83,11 +89,28 @@ function getRootPackageDirectory(loc: string) {
   }
 }
 
-// A bit of a hack: we keep this in local state and populate it
-// during the "prepare" call. Useful so that we don't need to pass
-// this implementation detail around outside of this interface.
-// Can't add it to the exported interface due to TS.
-let config: SnowpackConfig;
+async function setupPackageRootDirectory(dir: string, lockfile: LockfileManifest) {
+  await mkdirp(dir);
+  if (lockfile.dependencies) {
+    await fs.writeFile(
+      path.join(dir, 'package.json'),
+      JSON.stringify({dependencies: lockfile.dependencies}),
+      'utf8',
+    );
+    // Check for "lockfileVersion" to guarentee that lockfile is in expected format (not legacy).
+    if (lockfile.lock && lockfile.lock.lockfileVersion) {
```

Comment:
Should we be checking for a specific version here, e.g. `lockfileVersion >= 2`?

Reply:
because we're the ones generating these, it's safe to say that they're always a version we supprt. I guess this is based on the idea that npm wouldn't ship a breaking change here, but I think that's a safe bet given how slow the adoption of npm versions is.